src/components/CreateProfile/CreateProfile.jsx

```diff
@@ -132,6 +135,9 @@ const ProfilePage = ({ classes }) => {
             popperMessage: 'Your changes have been saved.',
           });
         })
+        .then(() => {
+          setUserCreated(true);
+        })
         .catch((err) => {
           console.log(err);
```

Comment:
If a user can't be created because of an invalid username, this will be a 422 with a message. I think we want to show a separate message for that ?